TestProject/tests/output/PNGOutputTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Tests\Output;
+
+use GameOfLife\CellularAutomat\Board;
+use GameOfLife\Output\PNGOutput;
+use GameOfLife\Options\Getopt;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Tests whether \Output\PNGOutput works as expected.
+ */
+class PNGOutputTest extends TestCase
+{
+    protected $pngOutput;
+    protected $board;
+    protected $getOpt;
+
+    /**
+     * @return void
+     */
+    protected function setUp(): void
+    {
+        $this->board = new Board(5, 5);
+        $this->pngOutput = new PNGOutput();
+        $this->getOpt = $this->createMock(Getopt::class);
+
+        $this->currentDir = getcwd();
+        chdir("/tmp");
+        if (!is_dir("/tmp/imageOutput")) mkdir("/tmp/imageOutput");
+    }
+
+    /**
+     * @return void
+     */
+    protected function tearDown(): void
+    {
+        parent::tearDown();
+
+        unlink("imageOutput/pngImage-000.png");
+        unlink("imageOutput/referenceImage.png");
+        rmdir("imageOutput/");
+        chdir($this->currentDir);
+    }
+
+    /**
+     * Asserts that the two png-images of the board aren't empty, by setting a background- and cell-color for each board.
+     */
+    public function testChecksIfBoardsAreTheSameAndNotEmpty()
+    {
+        $this->pngOutput->startOutput($this->getOpt);
+        $this->pngOutput->outputBoard($this->board);
+        $this->pngOutput->finishOutput();
+
+        $pngImage = imagecreate(5, 5);
+        imagecolorallocate($pngImage, 0, 0, 0);
+        imagecolorallocate($pngImage, 255, 255, 255);
+        imagepng($pngImage, "imageOutput/referenceImage.png");
+
+        $currentPngImage = file_get_contents("imageOutput/pngImage-000.png");
+
+        $this->assertEquals($currentPngImage, file_get_contents("imageOutput/pngImage-000.png"));
+    }
+}
```

Comment:
Please also add a test with a board that is non-empty.
To make it easier: It is ok that you save a reference image from the current output of the png-output, and commit it to the repository (somewhere inside the tests directory) and compare the generated image against the reference image. This at least allows you to detect if the output of the png changes in the future.